### PR TITLE
chore: update angular to v10 stable

### PR DESCRIPTION
**Pull request by Jinnie**
Also a small fix on the core/package.json build:package script, as the version is now set explicitly.

Signed-off-by: Ivan Donchev <idonchev@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

Update angular version to stable 10.0.2

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "test:bundles": "source-map-explorer dist/test-bundles/webpack.bundle.js",
     "build": "npm-run-all -s build:tokens build:sass build:ts build:sass:min build:package build:api build:copy",
     "build:watch": "npm-run-all -p build:sass:watch build:ts:watch",
-    "build:package": "cpy '**/package.json' '../dist/core/' --cwd=./src --parents && cpy ./package.json ./dist/core/ && cpy ./README.md ./dist/core/; replace '0.0.0' $npm_package_version ./dist/core/package.json; replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js && node ./package.js",
+    "build:package": "cpy '**/package.json' '../dist/core/' --cwd=./src --parents && cpy ./package.json ./dist/core/ && cpy ./README.md ./dist/core/; replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js && node ./package.js",
```
Comment (gnomeontherun) [inline packages/core/package.json:21]: Revert this file, we need to talk about this since Core doesn't follow the same pattern as others and it shouldn't be part of this change.

File: packages/angular/package.json [hunk 2]
```diff
@@ -40,14 +40,14 @@
...
   "dependencies": {
-    "@angular/animations": "10.0.0-rc.4",
-    "@angular/common": "10.0.0-rc.4",
-    "@angular/compiler": "10.0.0-rc.4",
-    "@angular/core": "10.0.0-rc.4",
-    "@angular/forms": "10.0.0-rc.4",
-    "@angular/platform-browser": "10.0.0-rc.4",
-    "@angular/platform-browser-dynamic": "10.0.0-rc.4",
-    "@angular/router": "10.0.0-rc.4",
+    "@angular/animations": "10.0.2",
```
[the remark this answers is not in the document]
Comment (Jinnie) [reply packages/angular/package.json:43]: Done

File: packages/core/package.json (hunk 1 above)
Comment (Jinnie) [reply packages/core/package.json]: Done
